.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-exclude = .git,__pycache__,build,dist,versioneer.py,mongo/_version.py,docs/source/conf.py
+exclude = .git,__pycache__,build,dist,versioneer.py,mongo/_version.py,docs/source/conf.py,suitcase/mongo_layout2/*
```

Comment:
This can be removed because `mongo_layout2` is gone now and `mongo_embedded` in flake8-compliant.

Reply:
updated